storybook/stories/Components/Chart/pie.stories.tsx

```diff
@@ -39,6 +39,53 @@ export default {
 } as Meta;
 
 
-export const PieChartBaseStory = () => (
-    <PieChart title="Pie Chart Example" data={data} />
-);
\ No newline at end of file
+const PieChartStory = ({
+    storyTitle,
+    ...props
+}: {
+    storyTitle: string;
+    props: TPieChart;
+}) => (
+    <>
+        <h2>{storyTitle}</h2>
+        <div style={{width: '600px', height: '600px' }}>
+            <PieChart data={data} {...props} width={400} height={500} />
+        </div>
+    </>
+);
+
+
+export const PieChartBasic = PieChartStory.bind({});
+PieChartBasic.args = {
+    storyTitle: 'Pie Chart ',
```

Comment:
L'espace à la fin de `storyTitle` est nécessaire ?

```suggestion
    storyTitle: 'Pie Chart',
```

Reply:
non, mais c'est pas un problème non plus dans ce context.

Reply:
j'ai enlevé l'espace